docs/reference/modules/node.asciidoc

```diff
@@ -1,5 +1,5 @@
 [[modules-node]]
-== Node
+=== Node settings
```

Comment:
This topic is our "source of truth" for node definitions and is referenced in customer agreements. I agree we need a "node settings" topic, but having our official node info under "node settings" is confusing. As a follow-on, we need to rework this to split apart the settings from the definitions. IOW, merging to master is fine, but the content needs to be reworked before it becomes "current".

Reply:
I'm good with leaving this as `node`. I'm not actually sure it makes sense to split the definitions from the settings here as they're pretty connected.

Reverted with b008abb4a11